CONTRIBUTING.rst

```diff
@@ -36,5 +37,128 @@ Pull Request. Contributions may include:
 * Helping to onboard new contributors
 * Other related activities
 
+Development
+===========
 
-Check `README.rst <README.rst>`_ development section instructions.
\ No newline at end of file
+Requirements
+-------------
+
+- Python >=3.9
+- Pipenv
+
+Getting the source code
+-----------------------
+
+`Fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_ the
+repository on `GitHub <https://github.com/vmware/repository-service-tuf-cli>`_ and
+`clone <https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository>`_
+it to your local machine:
+
+.. code-block:: console
+
+    git clone git@github.com:YOUR-USERNAME/repository-service-tuf-cli.git
+
+Add a `remote
+<https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork>`_ and
+regularly `sync <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork>`_ to make sure
+you stay up-to-date with our repository:
+
+.. code-block:: console
+
+    git remote add upstream https://github.com/vmware/repository-service-tuf-cli
+    git checkout main
+    git fetch upstream
+    git merge upstream/main
+
+Preparing the environment
+-------------------------
+
+After installing Python, install the pipenv tool:
+
+.. code:: shell
+
+    $ pip install pipenv
+
+
+Create a virtual environment for this project:
+
+.. code:: shell
+
+    $ pipenv shell
+
+
+Install the requirements from the Pipfile.
+
+The flag -d will install the development requirements:
+
+.. code:: shell
+
+    $ pipenv install -d
+
+
+.. note::
+
+    macOS running on MacBook M1
+
+    For developers, after the above command, run:
```

Comment:
Not sure what this sentence means: `For developers, after the above command, run:`
Do we mean:
```suggestion
    For developers using macOS on MacBook M1 lapton, after the above command, run:
```

Also, what about the M2 based Mac laptops? Do they need that as well?
Do we need to say `all developers with non-intel chip-based Mac laptops need to run these commands as well?

Reply:
@kairoaraujo maybe you can help us here?
If it's a big discussion I suggest opening another issue, as per the description of this PR, it would be better to focus on the structure and only low-hanging fruit of the content.

Reply:
The choice of RST is for the Read The Docs Sphinx and some complex features, such as `include` with cutting parts, to avoid managing the exact text in multiple files.
In the umbrella repository, we compile all documentation together.

``note:`` is not supported by GH, but it is ok. Also, the Markdown doesn't have a proper 'note'.

Reply:
+1